arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -1819,7 +1819,6 @@
           "description": "Resource Id"
         },
         "name": {
-          "readOnly": true,
```

Comment:
..but are you accepting a **Resource** in your payload? Conceptually, you are changing how your resources are modeled (which affects **all** usages or Resource) in order to satisfy the serialization requirements for one api.